src/basics.md

```diff
@@ -194,23 +184,30 @@ fn main() {
 
 [ex-rand-dist]: #ex-rand-dist
 <a name="ex-rand-dist"></a>
-## Generate random numbers with normal distribution
+## Random number distributions
 
 [![rand-badge]][rand] [![cat-science-badge]][cat-science]
 
-Creates a [`Normal`] distribution with mean `3` and standard deviation `5`
-and generates a random value with [`IndependentSample::ind_sample`].
+You've already seen how to create numbers with uniform distribution [above][ex-rand-range]
+(using [`Range`]). Other distributions are used in the same way: create a distribution, then
```

Comment:
most of this description is obvious from the trivial code below no need to be verbose here.